cdk.context.json

```diff
@@ -30,37 +30,6 @@
       }
     ]
   },
-  "vpc-provider:account=368354993240:filter.tag:ApplicationLayer=networking:filter.tag:ApplicationName=geostore:region=ap-southeast-2:returnAsymmetricSubnets=true": {
```

Comment:
I would probably put the changes to this file in a separate PR, since commits are squashed when merging. That way, if we ever need to undo it's a trivial, single-commit revert.

Reply:
Have moved this to #1887